oeps/best-practices/oep-0019-bp-developer-documentation.rst

```diff
@@ -126,6 +126,7 @@ ADRs
      - Inspired by `Nygard's post`_, Architecture Decision Records (ADRs) are local lightweight documents of technical decisions co-located with their corresponding code. They provide rationale behind the code and the archeological path to their current state. The status of an ADR can be:
 
        * **draft** if the decision is still preliminary and in experimental phase
+       * **provisional** if the decision is still preliminary and in experimental phase
```

Comment:
**Note to Reviewer**: This line was added to the new ADR template in this file, but this section was not adjusted to match.